test/Microsoft.ML.TestFrameworkCommon/Utility/PathResolver.cs

```diff
@@ -1,10 +1,12 @@
 ﻿// Taken from https://raw.githubusercontent.com/mellinoe/nativelibraryloader/586f9738ff12688df8f0662027da8c319aee3841/NativeLibraryLoader/PathResolver.cs
```

Comment:
These changes seem to be causing some issues finding DLL's, especially on non-windows OS. Like [this](https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-machinelearning-refs-pull-6168-merge-08e0b5fe86a14e8499/Microsoft.ML.TimeSeries.Tests/1/console.0b33654d.log?sv=2019-07-07&se=2022-05-17T17%3A12%3A38Z&sr=c&sp=rl&sig=zzyOtZZQhMtEK8s59t%2B2AljfYDH6KVMOrz1Nx5rCxG0%3D) one tried to one some MKL tests, but it is on arm and should have skipped those tests. So something isn't quite right here.